hicommonwealth/commonwealth
# Added stake enabled status screen

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6689

## Description of Changes
- Added stake enabled status screen for communities that have stake enabled.

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_EXISTING_COMMUNITY_STAKE_INTEGRATION_ENABLED` in `.env`
- Visit `/:scope/manage/integrations/stake` page of any community 
  - where stake is enabled, verify you see the stake status page.
  - where stake cannot be enabled, verify you see the "Not found" page

## Deployment Plan
N/A

## Other Considerations
N/A
